packages/im-base/src/readers/Movie.js

```diff
@@ -0,0 +1,26 @@
+// Lodash
+import _property from "lodash/property";
+
+const id = _property("id");
+const title = _property("title");
+const name = _property("name");
+const releaseDate = _property("releaseDate");
```

Comment:
releaseDate is not an attribute for movie.
Add data for hotspots, overlays, triggers, editorId, publishedAt, createdAt etc.